src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));

```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/4a16fe28-5cec-4468-9365-18ea45a91178)

![image](https://github.com/mate-academy/jv-lottery/assets/20326858/2d63207c-b0e2-4d08-bc1e-4fb53629527a)

Reply:
The provided link is not working, please provide a working one.